src/store/statusStore.ts

```diff
@@ -24,7 +24,7 @@ export const fetchStatus = async (digitalTwinId: DtId) => {
     }
     let response;
     try {
-        response = await axios.get(location, { timeout: 2000 });
+        response = await axios.get(location, { timeout: 5000 });
```

Comment:
why ?

Reply:
time out wa groter maken voor te kijken of peer online is